### PR TITLE
Source fix

### PR DESCRIPTION
**Pull request by musselwhizzle**

### CODE REVIEW

**Comment by musselwhizzle** [comment]
@chrispix99 PR for you.

File: library/src/main/java/com/climate/mirage/Mirage.java [hunk 1]
```diff
@@ -165,14 +166,18 @@ public MirageRequest load(String uri) {
...
+		if (!TextUtils.isEmpty(scheme)) {
+			if (scheme.startsWith(SCHEME_FILE) ||
+					scheme.startsWith(SCHEME_ANDROID_RESOURCE)) {
+				r.diskCacheStrategy(DiskCacheStrategy.RESULT);
+			}
 		}
 		return r.mirage(this).uri(uri);
 	}
 
 	public MirageRequest load(File file) {
```
Comment (chrispix99) [inline library/src/main/java/com/climate/mirage/Mirage.java]: Looks good, thoughts about moving this load method next to the other one so the overloaded methods are in sequence?

**Review by chrispix99 — APPROVED** [review]
Looks good, one comment left no action required though.

File: library/src/main/java/com/climate/mirage/Mirage.java (hunk 1 above)
Comment (musselwhizzle) [reply library/src/main/java/com/climate/mirage/Mirage.java]: ok will move.

**Pull request merged**